### PR TITLE
Make short lines prettier

### PR DESCRIPTION
**Pull request by 2uasimojo**
Fix up a couple of aesthetic issues brought about by enforcing 80c
lines:
- Eliminate multiline conditions with some simple refactoring.
- Don't split multiline strings really close to the end.
- Use keys in a couple of the longer interpolated strings to make it
  easier to track which vars go where.

### CODE REVIEW

**Comment by 2uasimojo** [comment]
I still haven't figured out how to stack new commits onto an existing PR without squashing. Please review only commit 2d5c2f9a872d22edb42f4b9697ec969a259fce14 in this PR; commit def4dd47abc44d9c84a542ac874747eb693c81b7 should be reviewed in #47.

File: build/generate_syncset.py [hunk 1]
```diff
@@ -1,31 +1,37 @@
...
-
 import os
-import sys
 import argparse
 import copy
 
+import oyaml as yaml
+
+
+TPL = "00-osd-managed-cluster-validating-webhooks.selectorsyncset.yaml.tmpl"
```
Comment (jewzaam) [inline build/generate_syncset.py:10]: TPL sounds like a TLA, could we not shorten unless it's a very common shortening?  Recommend using `TEMPLATE` at a minimum.

**Comment by 2uasimojo** [comment]
> We (@jharrington22, @mwoodson, and me) discussed and are going to be looking at 120 char limit. Much of this PR is assuming a 80 char line limit and will not be necessary with 120 chars limit. Please sync w/ James on this.

Done via https://github.com/openshift/managed-cluster-validating-webhooks/pull/60 (it was easier to start over).

**Comment by 2uasimojo** [comment]
Superseded by #60

**Pull request closed without merging**